sdk/containerregistry/azure-containerregistry/azure/containerregistry/_models.py

```diff
@@ -318,13 +318,13 @@ def __init__(self, **kwargs):
         self.digest = kwargs.get("digest")
 
 
-class ManifestDigestValidationError(ValueError):
+class DigestValidationError(ValueError):
     """Thrown when a manifest digest validation fails.
     :param str message: Message for caller describing the reason for the failure.
     """
 
     def __init__(self, message):
         self.message = message
         super(  # pylint: disable=super-with-arguments
-            ManifestDigestValidationError, self
+            DigestValidationError, self
```

Comment:
can this just be `super().__init(self.message)` ? You can get rid of the pylint block as well.